__tests__/integration/core-p2p/__support__/mock-socket-server/broker.js

```diff
@@ -1,5 +1,5 @@
-const SCBroker = require('socketcluster/scbroker');
```

Comment:
just delete the whole file and remove the `brokerController` line from `index.js` (should have done it myself, missed that)